typescript/cli/src/deploy/warp.ts

```diff
@@ -393,10 +423,11 @@ export async function runWarpRouteApply(params: ApplyParams): Promise<void> {
   WarpCoreConfigSchema.parse(warpCoreConfig);
   const addresses = await registry.getAddresses();
 
-  // Convert warpCoreConfig.tokens[] into a mapping of { [chainName]: Config }
-  // This allows O(1) reads within the loop
-  const warpCoreByChain = Object.fromEntries(
-    warpCoreConfig.tokens.map((token) => [token.chainName, token]),
+  const tokenConfigsByChain = Object.fromEntries(
```

Comment:
not sure if i agree with "tokenConfig". arguably, there are 2 "tokenConfigs": WarpCore and WarpDeploy.